ci-operator/step-registry/cnv/deploy-odf/cnv-deploy-odf-commands.sh

```
@@ -0,0 +1,236 @@
+ODF_DEFAULT_STORAGE_CLASS="${ODF_STORAGE_CLUSTER_NAME}-ceph-rbd"
+# ODF_DEFAULT_STORAGE_CLASS=ocs-storagecluster-ceph-rbd-virtualization
+DEFAULT_STORAGE_CLASS=${DEFAULT_STORAGE_CLASS:-${ODF_DEFAULT_STORAGE_CLASS}}
+
+readonly ODF_CATALOG_IMAGE="quay.io/rhceph-dev/ocs-registry:latest-stable-${ODF_VERSION_MAJOR_MINOR}"
+readonly ODF_CATALOG_NAME=odf-catalogsource
+
+readonly CLUSTER_PULL_SECRETS_ORIGINAL=/tmp/ps1.json
+readonly CLUSTER_PULL_SECRETS_UPDATED=/tmp/ps.json
+readonly ODF_QUAY_CREDENTIALS_FILE=/tmp/secrets/odf-quay-credentials/rhceph-dev
+
+source <(curl -sL "https://raw.githubusercontent.com/openshift-cnv/cnv-ci/refs/heads/master/hack/shared-functions.sh")
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)